factory/ethElrondBridgeComponents.go

```diff
@@ -535,12 +533,12 @@ func (components *ethElrondBridgeComponents) createElrondToEthereumBridge(args A
 		TimeForTransferExecution: timeForTransferExecution,
 	}
 
-	components.elrondToEthBridge, err = ethElrond.NewBridgeExecutor(argsBridgeExecutor)
+	bridge, err := ethElrond.NewBridgeExecutor(argsBridgeExecutor)
 	if err != nil {
 		return err
 	}
 
-	components.elrondToEthMachineStates, err = elrondToEthSteps.CreateSteps(components.ethToElrondBridge)
+	components.elrondToEthMachineStates, err = elrondToEthSteps.CreateSteps(bridge)
```

Comment:
here was the bug, instead of using `components.elrondToEthBridge` we used the other one: `components.ethToElrondBridge`